web/templates/user_display_only_pill.hbs

```diff
@@ -6,7 +6,7 @@
         <span class="pill-label">
             <span class="pill-value">{{display_value}}</span>
             {{#unless is_active}}
-                <i class="fa fa-ban deactivated-user-icon tippy-zulip-delayed-tooltip" data-tippy-content="{{#if is_bot}}{{t 'Bot is deactivated' }}{{else}}{{t 'User is deactivated' }}{{/if}}"></i>
+                <i class="fa fa-ban pill-deactivated deactivated-user-icon tippy-zulip-delayed-tooltip" data-tippy-content="{{#if is_bot}}{{t 'Bot is deactivated' }}{{else}}{{t 'User is deactivated' }}{{/if}}"></i>
```

Comment:
The pill design here is different from what we have for other places, in which there is some padding on both the sides. So, instead of this fix, might be better to add a commit to just make that pill consistent with other pills on the page.

Reply:
| Before | After( made consistent with pill design) |
| -------- | ------- |
| ![image](https://github.com/zulip/zulip/assets/73663475/cb84f3cb-0d90-43fc-b9e8-2babe3e9db6b)  | ![image](https://github.com/zulip/zulip/assets/73663475/5fbbf2ed-7de4-4656-8307-06b58e0acb4d)  |